{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -2,8 +2,11 @@ language: python
 python: '3.5'
 sudo: false
 env:
+{% if cookiecutter.c_extension_support|lower == "yes" %}
```

Comment:
Ah, almost forgot (sorry). This needs to be always around (not fun then one off your dependencies falls flat and segfaults on you). Plus you can always segfafult the python interpreter from pure python code (bugs and other fine issues).